modules/server/src/main/ts/bedrock/core/ConsoleReporter.ts

```diff
@@ -1,9 +1,45 @@
-import { TestResults } from '../server/Controller';
+import * as chalk from 'chalk';
+import { highlight } from 'cli-highlight';
+import { TestErrorData, TestResults } from '../server/Controller';
 
 const numberOfErrorsToPrint = 5;
 
-export const generateReport = (data: TestResults): string => {
+const highlightDiff = (diff: string) =>
+  highlight(diff, { language: 'diff' });
```

Comment:
Is this `cli-highlight` library an arbitrary syntax highlighter for all languages? Are we okay with pulling that in

Reply:
No idea what you're asking here sorry?

Reply:
Discussed on slack. There are not many viable alternatives for this, so we're okay with this. If anyone finds anything, I'm more than happy to use something else.